Documentation/guides/DotNet5.md

```diff
@@ -120,4 +120,58 @@ The following instructions can be used for early preview testing.
 dotnet publish -t:Install *.csproj
 ```
 
-[0]:  https://github.com/dotnet/installer#installers-and-binaries
+## Package Versioning Scheme
+
+Our NuGet packages are versioned using [Semver 2.0.0][2].
```

Comment:
Another pet peeve of mine, but I would suggest *removing **all** mention* of SemVer.  It *can't* reasonably apply to us, as *we* have *no idea* when semantics breaks occur!

This is more noticeable in the "current world order" in which we bundle Mono, and Mono may import some blob of code from mono/referencesource, which then proceeds to break a bunch of previously expected *semantics*, e.g. https://docs.microsoft.com/en-us/xamarin/android/release-notes/10/10.0#mono-framework-version-update-to-64

Semantic compatibility is *hard*, and SemVer is *all about* ***semantic*** compatibility, and at a very *deep* level, we're so far removed from "semantically relevant" changes that we have *no* idea if some of our semantics have broken.

This might plausibly change in a .NET 5 world order, in that *we* are no longer redistributing the BCL, but I'm still very leery of the entire idea.